third_party/move/move-model/src/lib.rs

```diff
@@ -119,6 +119,11 @@ pub fn run_model_builder_in_compiler_mode(
             .set_verify(compile_verify_code)
             .set_keep_testing_functions(compile_test_code)
             .set_lang_v2(language_version != LanguageVersion::V1)
```

Comment:
If we're adding language_version, then let's get rid of lang_v2.

Reply:
done